dm/config/task_test.go

```diff
@@ -1032,6 +1033,48 @@ func (t *testConfig) TestTaskConfigForDowngrade(c *C) {
 	c.Assert(cfgForDowngrade, DeepEquals, cfgForClone)
 }
 
+func (t *testConfig) TestTlsTaskConfig(c *C) {
+	cfg := NewTaskConfig()
+	taskRowStr := `---
+name: test
+task-mode: all
+target-database:
+    host: "127.0.0.1"
+    port: 3307
+    user: "root"
+    password: "123456"
+    security:
+      ssl-ca: "testdata/ca.pem"
+      ssl-cert: "testdata/cert.pem"
+      ssl-key: "testdata/key.pem"
+block-allow-list:
+  instance:
+    do-dbs: ["dm_benchmark"]
+mysql-instances:
+  - source-id: "mysql-replica-01-tls"
+    block-allow-list: "instance"
+`
+
+	err := cfg.RawDecode(taskRowStr)
```

Comment:
I mean test RawDecode -> LoadTLSContent -> String -> Decode

Reply:
already tested in [here](https://github.com/Ehco1996/dm/blob/9a6e757cb3b5158111e7885764b098c8b535d7e4/dm/config/task_test.go#L1071)

Reply:
that's RawDecode -> LoadTLSContent -> adjust-> String -> Decode

Reply:
fixed in https://github.com/pingcap/dm/pull/1910/commits/4afcc9309b787c3e5ddb5c1f16b039eedaab7d35